### ISSUE TITLE
JetEngine: Forms - Custom item template is not displayed when the value from meta field is set

### ISSUE THREAD

**Issue by poziomapl** [CrocoBlock/suggestions#3234]
**Describe the bug**
I am creating a checkbox form.
It gets data from Terms and the Colors taxonomy (which I created with JetEngine). 
Then I select the custom item template display and select the previously created listing template as a template. 
This configuration works fine and the listing displays nicely, but the **problem appears when I set the value from meta field**.

I have to complete it because by default, when an email is sent, the post id is returned in the form of a number. Instead of getting a number like 23, 44, I would like a color name.

For this purpose there is a field: "Value from meta field:" but when I choose a meta field with the name Listing grid is not displayed in the form.

When I uncheck the Custom item template, a regular checkbox is displayed and the value is returned correctly.

**To Reproduce**
1. Create a form with a checkbox field type.
2. Select data extraction from Terms and Taxonomy
3. Create a Listing for Terms and Taxonomies
4. In the form, set the created listing as a template for displaying checkbox fields.
5. Set / fill in the "Value from meta field" field.
6. Apply the changes and display the form on the frontend.
7. Checkbox with Listing template is not displayed.
8. Clear the Value from meta field and save the changes.
9. Checkbox with template displays correctly.

**Expected behavior**
When I set the "Value from meta field" checkbox, it displays as a listing from the selected template, and in the sent email instead of post id I get the value that I set in the value from meta field.

**Screenshots**
No screenshoot needed.

**Desktop (please complete the following information):**
 - OS: macOS Big Sur latest
 - Browser: Chrome latest
 - Version: JetEngine 2.7.8

**Additional context**
Maybe I'm doing something wrong and that's why there is a problem. But maybe it is possible not to enter the meta field, but retrieve the data for the value, e.g. from the created tag (color in my case).

Thank you very much for your help, hint or solution to the problem. Your team is really cool and you are doing an amazing job! 🙂

**Comment by Floydtm** [comment]
It is not entirely clear to me why you are doing this, and it seems to me that such a case will not work correctly. You'd better contact our support team, the guys will help you set up everything. Thank you!

**Comment by poziomapl** [comment]
I have created a post type with my products.
For these products, I also created taxonomy with colors.
Colors in the offer I have about 60. Selecting them from the Select list or 60 Checxboxes is bad in terms of UX...

(I will add that I create a form for the client where he can choose in what color he wants to get a product. 
And I receives information with the form via mail.)

In form, there is the Custom Template Item option to display this as Listing Grid - I think so...

But when I use Custom Template item and set a field with a value from met field to receive the color name and not post ID in an email, then the listing does not display... So i cannot use listing template only a traditional checkbox...

And this is a problem ... I do not know why this thread has been closed ... I am asking for re-consider this thread.
